packages/connection/src/class-manager.php

```diff
@@ -2139,7 +2139,7 @@ public function get_access_token( $user_id = false, $token_key = false, $suppres
 
 		if ( $user_id ) {
 			if ( ! $user_tokens ) {
-				return $suppress_errors ? false : new \WP_Error( 'no_user_tokens' );
+				return $suppress_errors ? false : new \WP_Error( 'no_user_tokens', 'No user tokens found' );
```

Comment:
It's really cheap to wrap strings in translations, even if they're not shown right now they might be later.

Reply:
done! I also added i18n to all the error messages in that function